src/manage_nsfs/manage_nsfs_events_utils.js

```diff
@@ -347,4 +347,41 @@ NoobaaEvent.LOGGING_FAILED = Object.freeze({
     state: 'DEGRADED',
 });
 
+/////////////////////////////////////
+// CONFIG DIRECTORY UPGRADE EVENTS //
+/////////////////////////////////////
+
+NoobaaEvent.CONFIG_DIR_UPGRADE_STARTED = Object.freeze({
+    event_code: 'config_dir_upgrade_started',
+    entity_type: 'NODE',
+    event_type: 'INFO',
+    message: 'Config directory upgrade started.',
+    description: 'Config directory upgrade started.',
+    scope: 'NODE',
+    severity: 'INFO',
+    state: 'HEALTHY',
+});
+
+NoobaaEvent.CONFIG_DIR_UPGRADE_SUCCESSFUL = Object.freeze({
```

Comment:
I didn't see the use of this (`NoobaaEvent.CONFIG_DIR_UPGRADE_SUCCESSFUL`) and (`NoobaaEvent.CONFIG_DIR_UPGRADE_FAILED`) will it be in another PR?

Reply:
that's the beauty :)
these events are created on throw_cli_error() and on write_stdout_response(), both functions map the response code/error to events and creates them.

Reply:
Thanks, where can I see this mapping in the code?

Reply:
I added to these maps in the code, for example NSFS_CLI_SUCCESS_EVENT_MAP

Reply:
not sure what you are showing - 
https://github.com/noobaa/noobaa-core/pull/8670/files#diff-09cddef059e180f22a4f9a06578df9994e497b4acea47acfe323d81b514f1288R158-R159